lib/passport-saml/saml.js

```diff
@@ -1272,7 +1272,7 @@ SAML.prototype.generateServiceProviderMetadata = function( decryptionCert, signi
 
   if (this.options.logoutCallbackUrl) {
     metadata.EntityDescriptor.SPSSODescriptor.SingleLogoutService = {
-      '@Binding': 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST',
+      '@Binding': 'urn:oasis:names:tc:SAML:2.0:bindings:'+ this.options.singleLogoutServiceBinding || 'HTTP-POST',
```

Comment:
It seems we should avoid string concatenation here and just require the entire value to be set. That makes the documentation clearer and makes the configuration value in whatever config file much more explicit, which is good for a security system.

Reply:
Agreed.